docs/sources/administration/data-source-management/teamlbac/configure-teamlbac-for-tempo/index.md

```diff
@@ -0,0 +1,118 @@
+---
+title: "Configure LBAC for Tempo or Cloud Traces"
+description: "Use label-based access control (LBAC) to restrict Cloud Traces data by team and attribute rules."
+keywords:
+  - tempo
+  - datasource
+  - team
+labels:
+  products:
+    - cloud
+title: Configure LBAC for data sources for Tempo
+weight: 350
+_build:
+  list: false
+noindex: true
+---
+
+# Configure label-based access control for Tempo or Cloud Traces
+
+Label-based access control (LBAC) for Cloud Traces or Tempo lets you restrict which spans users and teams can access by defining rules using trace attributes. LBAC provides fine-grained, team-based access control within a single tenant and mirrors the experience used for logs and metrics LBAC in Grafana Cloud.
+
+Grafana uses the term LBAC as an umbrella term for label-based access control for all data sources.
+Traces use **attributes**, not labels, for access control, but the Grafana UI surfaces this functionality as LBAC for consistency.
+
+## How LBAC works
+
+When a user queries tracing data source, Grafana evaluates the user’s team memberships and the LBAC rules assigned to those teams. These rules are added to the request so the Tempo or Cloud Traces data source returns only permitted spans or attributes.
```

Comment:
we need to be extra explicit that this is a grafana cloud traces feature only - it will not exist in tempo oss nor GET